doc/connectivity/networking/api/wifi.rst

```diff
@@ -28,3 +28,36 @@ API Reference
 *************
 
 .. doxygengroup:: wifi_mgmt
+
+.. _wifi_shell_test_certs:
+X.509 Certificate header generation
+###################################
+
+Wi-Fi enterprise security requires use of X.509 certificates, test certificates
+in PEM format are committed to the repo and the below steps to be followed to convert
+them to C header file that can be included in Wi-Fi shell module.
+
+Convert CA format from PEM to DER
+=================================
+To convert the CA format from PEM to DER, use the following command:
+.. code-block:: none
+openssl x509 -outform der -in cas.pem -out ca.der
```

Comment:
This needs a specific format:
```suggestion
.. code-block:: none

     openssl x509 -outform der -in cas.pem -out ca.der
```
See for an example https://raw.githubusercontent.com/zephyrproject-rtos/zephyr/main/samples/net/wifi/README.rst